CMakeLists.txt

```diff
@@ -1,3 +1,26 @@
 cmake_minimum_required(VERSION 2.8)
 add_subdirectory(generator)
-add_subdirectory(indexgenerator)
\ No newline at end of file
+add_subdirectory(indexgenerator)
+
+INSTALL(
+  DIRECTORY data
+  DESTINATION ${CMAKE_INSTALL_PREFIX}/share/woboq_codebrowser
```

Comment:
I think it's reasonable to install the `data`. Most of the time `make install` is used for creating a distro package(such as rpm deb...), and the end user doesn't neccessarily have the source code tree. What if to install `data` as a tarball instead of the folder? Is that a better solution?